asllib/ASLTypingReference.mld

```diff
@@ -1041,30 +1078,92 @@ Formally, one of the following applies:
   {3 TypingRule.PGeq}
 
     {4 Rule}
+    All of the following applies:
+    - [p] is the pattern which matches anything greater than or equal to an expression [e];
+    - [t_e, e'] is the result of annotating [e] in [env]; 
+    - [e'] is pure;
```

Comment:
FIY, the LRM asks for a stronger check: that `e` (or `e'` similarly) is a compile-time constant.